src/utils/oauth2_util.py

```diff
@@ -76,3 +127,59 @@ def encrypt_password(password):
     hash_password = bcrypt.hashpw(password_bytes, salt)
     decode_hash_password = hash_password.decode("utf-8")
     return decode_hash_password
+
+
+def validate_scopes(scopes, secured_http_request_url):
+
+    endpoint_roles_permission = get_endpoint_roles_scopes_access_permission(
```

Comment:
Rename to `endpoint_permissions  = get_endpoint_permissions`

Reply:
done.